doc/release-process.md

```diff
@@ -12,10 +12,34 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+We aim to cut a regular release every 3-4 months, approximately twice as frequent as major Bitcoin Core releases. Every second release should be published one month before the feature freeze of the next major Bitcoin Core release, allowing sufficient time to update the library in Core.
+
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib/ -l secp256k1 && ./ecdsa
```

Comment:
```suggestion
gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib/ -static -l secp256k1 && ./ecdsa
```

Reply:
We could test the `libsecp256k1.pc` file as well:
```suggestion
gcc -o ecdsa examples/ecdsa.c $(PKG_CONFIG_PATH=$dir/lib/pkgconfig pkg-config --cflags --libs libsecp256k1)
```